py3status/modules/mpd_status.py

```diff
@@ -92,14 +103,17 @@ class Py3status:
     cache_timeout = 2
     format = '{state} [[[{artist}] - {title}]|[{file}]]'
     hide_when_paused = False
+    hide_when_playing = False
```

Comment:
`python-mpd2` indented. Maybe. Just filling in a potential missing option even if there is a chance that 99% of the users does not want it. Just making it well-around.